tips/TIP-0027/tip-0027.md

```diff
@@ -0,0 +1,320 @@
+
+The standard is defined utilising the creation mechanism for NFTs.
+
+`issuerId` is already defined in the framework, allowing every NFT created from the same source to be easily defined.
+
+Each NFT in the IOTA Tokenization Framework has its own unique address, that allows the ability to define a **collection UTXO** that can subsequently mint each unique NFT within that collection.
+
+The `nftId` of a collection NFT is defined as the `policyId`.
+
+the policyId will act as a unique identifier for the collection and would allow the `policyNft` to control the NFTs in a collection. This allows for unprecedented amounts of control where you can lock a collection for some time. Transfer the ownership of an entire collection via transfer of the `policyNft`.
+
+A creator should define the UTXO of their collection NFT as the sole minting source of an entire collection that is the 'policyId'.
+
+A creator may choose to burn the collection NFT on completion of minting or retain the collection NFT to add further NFTs to the collection over time.
+
+The UTXO of the collection NFT acts as the `policyId` for the collection and can be used in dApps to define the verified origin of an NFT.
+
+To call the defined `policyId` you should request the `policyId` UTXO of the collection NFT and resolve to the Policy Registry for human identifiable verification.
+
+### Policy Registry
+
+An open-source human identifiable registry of UTXO `policyId`s will be established to allow individuals the ability to support API integrations into dApps and websites that can support the connection of human identifiable names to `policyId`s.
+
+A creator should register their `policyId` UTXO to a human identifiable value (`policyName`) as a defined unique and verified collection.
+
+Reference to their `policyId` should be publicly visible on an official website or social media page.
+
+A `policyName` must be a unique string.
+
+An open-source committee must humanly verify the collection input to the registry.
+
+The Policy Registry has to be kept off-chain, but a commitment to its current state has to be stored in an output.
+
+An API to support the integration of the policy registry into dApps can be called from the open-source repository.
+
+When submitting a `policyId` to the registry the following keys should be defined.
+
+- `issuerId` must be the original issuerId UTXO string
+- `policyId` must be the unique UTXO string of the policyNFT
+- `policyName` must be a unique human identifiable string
+- `websiteUrl` must be an official website or social media page url that illustrates publicly the `policyId`
+
+```json
+{
+  "registration": {
+    "issuerId": "s345d6f7g8h9ji07gy8h99",
+    "policyId": "a3s45dr6ft7gy87hu98ji09",
+    "policyName": "My iNFT Collection",
+    "websiteUrl": "https://mywebsite.com/policyid"
+  }
+}
+```
+
+### Creator Royalties
```

Comment:
Does it make sense to include royalties in an NFT standard? From our side, we would like to support royalties, but there are scenarios where it is necessary to have them not immutable, or it is not enough to represent them as a percentage of a sales price traded against something that is not a currency.